### ISSUE TITLE
Cock ups!

### ISSUE THREAD

**Issue by sorinapuiac** [sorinapuiac/workshop-ratings#1]
- The title shoud be a h1
- The div with class form should be a form or should have role form and the inputs should be real inputs with proper labels
- div errors should have aria-hidden true or be missing; they should be announced using aria-live
- you can't navigate by tabs
- the starts should have content
  -the submit button should be a button or having the role="button"
- the button doesn't pass the contrast ratio test
- you should be notify that this content has been changed "You rated x stars" (using aria-live)
  -inappropriate markup for the starts
  -the font type for 'labels'
- ratings announced as "Black Star"
- Ratings widget is not keyboard navigable
- Submit action not possible by keyboard
  -"Rating" should be 'Your rating'
